docs/dev.rst

```diff
@@ -428,107 +552,43 @@ this code and it has to be tested manually. You can do something like this:
-
-.. code-block:: shell
-
-   $ docker compose exec oidcprovider /code/manage.py createuser yourname yourpassword yourname@example.com
-
-
-Giving users permission to upload symbols
------------------------------------------
-
-The user should write up a bug. See :ref:`upload-basics`.
-
-If the user is a Mozilla employee, needinfo the user's manager and verify the
-user needs upload permission.
+System tests
+------------
 
-If the user is not a Mozilla employee, find someone to vouch for the user.
+System tests are located in the repository in ``systemtests/``. See the
+``README.rst`` there for usage.
 
-Once vouched:
+System tests can be run against any running environment:
 
-1. Log in to `<https://symbols.mozilla.org/users>`_
-2. Use the search filter at the bottom of the page to find the user
-3. Click to edit and make give them the "Uploaders" group (only).
-4. Respond and say that they now have permission and should be able to either
-   upload via the web or create an API Token with the "Upload Symbol Files"
-   permission.
-5. Resolve the bug.
+* local: local dev environment
+* stage: the stage server environment
+* prod: the prod server environment--will not run destructive tests
 
+System tests can help verify that upload API and download API work. They
+periodically need to be updated as symbols files expire out of the systems.
 
-Viewing all metrics keys
-------------------------
 
-In the Symbols Service webapp, to get insight into all metrics keys that are
-used, a special Markus backend is enabled called
-``tecken.libmarkus.LogAllMetricsKeys``. It's enabled by default in local
-development. And to inspect its content you can either open
-``all-metrics-keys.json`` directly (it's git ignored) or you can run:
+Load tests
+----------
 
-.. code-block:: shell
+At various points, we've done some load testing of the system. The scripts are
+located in:
 
-   $ make shell
-   app@xxx:/app$ ./bin/list-all-metrics-keys.py
+https://github.com/mozilla-services/tecken-loadtests/
 
-Now you can see a list of all keys that are used. Take this and, for example,
-make sure you make a graph in Datadog of each and everyone. If there's a key in
-there that you know you don't need or care about in Datadog, then delete it
-from the code.
+They're generally unmaintained, but can be a good starting point for a new load
+testing effort.
 
-The file ``all-metrics-keys.json`` can be deleted any time and it will be
-recreated again.
 
 
-Localstack (S3 mock server)
----------------------------
 
-When doing local development we, by default, mock AWS S3 and instead use
-`localstack`_. It's an S3 emulator.
 
-When started with docker, it starts a web server on ``:4566`` that you can
-use to browse uploaded files. Go to ``http://localhost:4566``.
-
-.. _localstack: https://github.com/localstack/localstack
+How to do local Upload by Download URL
+======================================
 
+.. Note::
 
-How to do local Upload by Download URL
```

Comment:
I left this section here. It probably needs to be updated, but I don't want to spend the time on it right now. The systemtests have a download-by-upload test that looks at symbols.zip files from recent Taskcluster jobs and that's probably sufficient, so maybe we should just delete this section altogether.

https://github.com/mozilla-services/tecken/blob/ec4b677c8cbfe4705aeb0974d96958c677df64ae/systemtests/test_env.sh#L66-L69

Which would you prefer?